src/mpu6886.toit

```diff
@@ -80,78 +81,78 @@ class Driver:
       if tries == 0: throw "INVALID_CHIP"
       sleep --ms=1
 
-    reset  // reset
-    sleep --ms=100  // sleep 100ms for stable
-    init   // init sequence
+    reset
+    sleep --ms=100  // Sleep 100ms to stabilize.
+    init
 
-  /*
-   Reset
+  /**
+  Resets the chip.
   */
   reset:
-    reg_.write_u8 _PWR_MGMT_1 0b10000000  // software reset
+    reg_.write_u8 PWR_MGMT_1_ 0b10000000  // Software reset.
 
-  /*
-    WHO AM I
+  /**
+  Returns the value of the WHO_AM_I register.
+  The MPU-6886 should return 0x19.
   */
   whoami -> int:
-    """ Value of the whoami register. """
-    return (reg_.read_u8 _WHO_AM_I)
+    return reg_.read_u8 WHO_AM_I_
 
-  /*
-   Init sequence.
+  /**
+  Initializes the chip.
   */
   init:
-    reg_.write_u8 _PWR_MGMT_1     0x00
+    reg_.write_u8 PWR_MGMT_1_     0x00
     sleep --ms=10
-    reg_.write_u8 _PWR_MGMT_1     0x01 << 7
+    reg_.write_u8 PWR_MGMT_1_     0x01 << 7
     sleep --ms=10
-    reg_.write_u8 _PWR_MGMT_1     0x01
+    reg_.write_u8 PWR_MGMT_1_     0x01
     sleep --ms=10
-    reg_.write_u8 _ACCEL_CONFIG_1 0x10  // +- 8G
+    reg_.write_u8 ACCEL_CONFIG_1_ 0x10  // +- 8G
     sleep --ms=10
-    reg_.write_u8 _GYRO_CONFIG    0x18  // +- 2000dps
+    reg_.write_u8 GYRO_CONFIG_    0x18  // +- 2000dps
     sleep --ms=10
-    reg_.write_u8 _CONFIG         0x01
+    reg_.write_u8 CONFIG_         0x01
     sleep --ms=10
-    reg_.write_u8 _SMPLRT_DIV     0x05
+    reg_.write_u8 SMPLRT_DIV_     0x05
     sleep --ms=10
-    reg_.write_u8 _INT_ENABLE     0x00
+    reg_.write_u8 INT_ENABLE_     0x00
     sleep --ms=10
-    reg_.write_u8 _ACCEL_CONFIG_2  0X00
+    reg_.write_u8 ACCEL_CONFIG_2_  0X00
     sleep --ms=10
-    reg_.write_u8 _USER_CTRL      0x00
+    reg_.write_u8 USER_CTRL_      0x00
     sleep --ms=10
-    reg_.write_u8 _FIFO_EN        0x00
+    reg_.write_u8 FIFO_EN_        0x00
     sleep --ms=10
-    reg_.write_u8 _INT_PIN_CFG    0x22
+    reg_.write_u8 INT_PIN_CFG_    0x22
     sleep --ms=10
-    reg_.write_u8 _INT_ENABLE     0x01
+    reg_.write_u8 INT_ENABLE_     0x01
     sleep --ms=100
-  
-  /*
-    Acceleration
+
+  /**
+  Reads the acceleration.
```

Comment:
It would be nice to add information on what units are returned.

I have used m/s^2 for the lms303d. Not sure if we used the same value in our other drivers, though. (A bit on my TODO list to make sure everyone agrees on the units).
https://github.com/toitware/toit-lsm303d/blob/main/src/accelerometer.toit#L12

Similarly, switched to math.Point3f, as this is what we return most of the time for 3-axis measurements.